pkg/agent/route/route_windows.go

```diff
@@ -49,20 +50,23 @@ var (
 )
 
 type Client struct {
-	nodeConfig     *config.NodeConfig
-	networkConfig  *config.NetworkConfig
-	hostRoutes     *sync.Map
-	fwClient       *winfirewall.Client
-	bridgeInfIndex int
-	noSNAT         bool
-	proxyAll       bool
+	nodeConfig      *config.NodeConfig
+	networkConfig   *config.NetworkConfig
+	nodeRoutes      *sync.Map
+	serviceRoutes   *sync.Map
```

Comment:
I didn't find functions to load route entry from `serviceRoutes`, all calls are for store or delete. Is this cache really needed?

Reply:
Removed

Reply:
Add comments to explain what is stored in `serviceRoutes`?